stdlib/3.4/statistics.pyi

```diff
@@ -0,0 +1,44 @@
+# Stubs for statistics
+# See: http://docs.python.org/3/library/statistics.html
+
+import sys
+from typing import Iterable, Iterator, Sequence, TypeVar, Union, overload
+from decimal import Decimal
+
+# Note: according to the docs, statistics only explicitly supports
+# int, float, Decimal, and Fraction. Other types within the numeric
+# tower are not supported. It also states mixing together different
+# types results in undefined behavior, so the type signatures below
+# deliberately enforce that numeric types may not be mixed.
+#
+# TODO: Once either https://github.com/python/typeshed/pull/545 or
+# https://github.com/python/typeshed/pull/94 is accepted, add support
+# for fractions.
+_TNum = TypeVar('_TNum', int, float, Decimal)
+_T = TypeVar('_T')
+
+def mean(data: Union[Iterator[_TNum], Sequence[_TNum]]) -> _TNum: ...
```

Comment:
I worry that this makes mypy think that the return type for `mean([1, 2, 4])` is an int. It actually returns a float. Similarly, it allows mixing ints with Decimal (but not float with Decimal). I think the only way to express this reasonably may be a complex set of overloads...